db/models/feedback.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Feedback = db.define('feedback', {
```

Comment:
Feedback for...what, exactly? (And does there need to be a "feedback for" field?)